plugins/utils/hydra/default.nix

```diff
@@ -38,7 +38,7 @@ helpers.neovim-plugin.mkNeovimPlugin config {
 
   callSetup = false;
   extraConfig = cfg: {
-    extraConfigLua = ''
+    plugins.hydra.config.init = ''
```

Comment:
Have you replaced `extraConfigLua` assignments in _all_ mkNeovimPlugin call sites treewide? :eyes:

Reply:
To my knowledge yes!